lib/models/ObjectQueueEntry.js

```diff
@@ -193,21 +194,24 @@ class ObjectQueueEntry extends ObjectMD {
         return this.clone()
             .setAccountId(this.getAccountId())
             .setReplicationSiteStatus(site, 'COMPLETED')
-            .setReplicationStatus(this._getGlobalReplicationStatus());
+            .setReplicationStatus(this._getGlobalReplicationStatus())
+            .setOriginOp('');
```

Comment:
should we not set a value anyway?
it could (eventually) be used to generate non-standard notification (or make it easier/safer to refactor), and will be helpful already for troubleshooting: as we can directly see what caused the metadata change...